kolibri/plugins/learn/assets/src/views/total-points/index.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+
+  <div v-if="totalPoints" class="points" ref="points">
+    <div class="circle in-points">
+      <points-icon class="icon" :active="true"/>
+    </div>
+    <span class="total in-points">{{ $formatNumber(totalPoints) }}</span>
+    <ui-tooltip trigger="points" :position="'bottom right'" :openOn="'hover focus'">{{ $tr('totalPoints', { points: totalPoints }) }}</ui-tooltip>
+  </div>
+
+</template>
+
+
+<script>
+
+  const { totalPoints } = require('kolibri.coreVue.vuex.getters');
+  const { fetchPoints } = require('kolibri.coreVue.vuex.actions');
+
+  module.exports = {
+    $trNameSpace: 'totalPoints',
+    $trs: {
+      totalPoints: 'You have earned { points, number } points!',
+    },
+    components: {
+      'points-icon': require('kolibri.coreVue.components.pointsIcon'),
+      'ui-tooltip': require('keen-ui/src/UiTooltip'),
```

Comment:
I am not certain that it is. And while it is meant to pop up on focus, when I keyboard navigated to the element, it failed to appear. Will probably need some work.